src/nplinker/metabolomics/gnps/gnps_file_mapping_loader.py

```diff
@@ -13,14 +15,22 @@
 
 class GNPSFileMappingLoader(FileMappingLoaderBase):
 
-    def __init__(self, filename: str):
-        self._filename: str = filename
+    def __init__(self, filepath: str | PathLike):
```

Comment:
```suggestion
    def __init__(self, file: str | PathLike):
```
we decided to use the name `file` to refer to the path of a file